applications/ecallmgr/src/ecallmgr_fs_node_sup.erl

```diff
@@ -31,11 +31,16 @@
 
 -include("ecallmgr.hrl").
 
--define(CHILDREN, [<<"node">>, <<"authn">>, <<"route">>, <<"channel">>
-                  ,<<"config">>, <<"resource">>, <<"notify">>
+-define(CHILDREN, [<<"authn">>
+                  ,<<"channel">>
                   ,<<"conference">>
+                  ,<<"config">>
                   ,<<"event_stream_sup">>
                   ,<<"msg">>
+                  ,<<"node">>
+                  ,<<"notify">>
+                  ,<<"resource">>
+                  ,<<"route">>
```

Comment:
AFAIK none of the supervisors rely on the others being up yet. It is a one-for-one supervisor so they should operate independently anyway.